src/rest-api/endpoints/management-endpoint/elasticsearch-endpoint.md

```diff
@@ -177,14 +158,16 @@ POST /management/elasticsearch/optimize
 
 ### Sample
 
-To optimize the `default` repository:
+To optimize the `default` index repository:
```

Comment:
the `default` repository index? Aka the index of the repository named `default` right?